administrator/components/com_admin/sql/updates/postgresql/4.0.0-2020-02-20.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "#__content_types" ALTER COLUMN "table" TYPE varchar(2048);
```

Comment:
```suggestion
ALTER TABLE "#__content_types" ALTER COLUMN "table" TYPE character varying(2048);
```
Has to be done like this because our database checker is so silly, see [https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/src/Schema/ChangeItem/PostgresqlChangeItem.php#L146](https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/src/Schema/ChangeItem/PostgresqlChangeItem.php#L146).